### PR TITLE
refactor: Update the import automation example plugin to match the documentation

### PR DESCRIPTION
**Pull request by daniel-schroeder-zeiss**

### CODE REVIEW

File: examples/SimpleGeneratorPlugin/AutomationConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+/* Carl Zeiss Industrielle Messtechnik GmbH        */
+/* Softwaresystem PiWeb                            */
+/* (c) Carl Zeiss 2024                             */
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#endregion
+
+using Zeiss.PiWeb.Sdk.Import.Modules.ImportAutomation;
+
+namespace SimpleGeneratorPlugin
```
Comment (czdietrich) [inline examples/SimpleGeneratorPlugin/AutomationConfiguration.cs]: Does it make sense to use file scoped namespace here?

File: examples/SimpleGeneratorPlugin/ImportRunner.cs [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+                _ActivityService.PostActivityEvent(EventSeverity.Error, "The import target is not supported");
+                return;
+            }
+
+            try
+            {
+                var uri = new Uri(_ImportTarget.ServiceAddress);
+                var authData = _ImportTarget.AuthData;
+                using var restClient = CreateDataServiceClient(uri, authData);
+                var random = new Random();
```
Comment (czdietrich) [inline examples/SimpleGeneratorPlugin/ImportRunner.cs]: There is also `Random.Shared` that can be used out of the box.

**Review by czdietrich — APPROVED** [review]

File: examples/PiWebImportSdkExamples.sln [hunk 3]
```diff
@@ -1,54 +1,60 @@
...
+# Visual Studio Version 17
+VisualStudioVersion = 17.10.35122.118
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FirstImportAutomation", "FirstImportAutomation\FirstImportAutomation.csproj", "{5A543501-5133-4572-AEDD-02F53376B6B7}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SecondImportAutomation", "SecondImportAutomation\SecondImportAutomation.csproj", "{C4A3E62B-59DC-4CAD-9B95-B749954C6C30}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "StartingAPlugin", "StartingAPlugin\StartingAPlugin.csproj", "{F24BEAEE-81F7-4A28-922E-4D10E3EE1CAA}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Projektmappenelemente", "Projektmappenelemente", "{5AA887A8-68D2-42F6-9825-E7553F2C40CC}"
```
Comment (czdietrich) [inline examples/PiWebImportSdkExamples.sln]: `Projektmappenelemente` should be renamed to `SolutionItems` I guess.

File: examples/SimpleGeneratorPlugin/ImportRunner.cs (hunk 2 above)
Comment (daniel-schroeder-zeiss) [reply examples/SimpleGeneratorPlugin/ImportRunner.cs]: From what I understand, Random.Shared goes the extra mile to ensure thread safety (hence the name). I don't think it is supposed to be used in a non-concurrent context.

File: examples/SimpleGeneratorPlugin/AutomationConfiguration.cs (hunk 1 above)
Comment (daniel-schroeder-zeiss) [reply examples/SimpleGeneratorPlugin/AutomationConfiguration.cs]: Yes I'll change the namespaces.

File: examples/SimpleGeneratorPlugin/ImportRunner.cs (hunk 2 above)
Comment (czdietrich) [reply examples/SimpleGeneratorPlugin/ImportRunner.cs]: That's true, indeed. I think the main idea is to have a single global instance to allocate only once. 
I have no problem if you prefer to instantiate a new `Random` here and leave it as is.

File: examples/PiWebImportSdkExamples.sln (hunk 3 above)
Comment (daniel-schroeder-zeiss) [reply examples/PiWebImportSdkExamples.sln]: Done.

**Pull request merged**